### DISCUSSION TITLE
hiding console without closing stdout

### DISCUSSION THREAD

**Discussion by sancelot** [pyinstaller/pyinstaller#6441] [PyInstaller]
Hi,
I would need to hide the console , without closing stdout / stderr .

Otherwise there will be some application issues.

Is it possible ???

**Comment by rokm** [comment]
You can programmatically hide console window using [`GetConsoleWindow`](https://docs.microsoft.com/en-us/windows/console/getconsolewindow) and [`ShowWindow`](https://docs.microsoft.com/en-us/windows/win32/api/winuser/nf-winuser-showwindow) functions from win32 API. 

You can use `pywin32` bindings (its `win32console` and `win32gui` modules, respectively), or you can probably also bind both functions directly using`ctypes`.
